test/parallel/test-stream-readable-destroy.js

```diff
@@ -160,3 +160,18 @@ const { inherits } = require('util');
 
   new MyReadable();
 }
+
+{
+  // destroy and destroy callback
+  const read = new Readable({
+    read() {}
+  });
+  read.resume();
+
+  const expected = new Error('kaboom');
+
+  // the callback will be called asynchronously
```

Comment:
If this statement is important you should assert it (set a field on `expected` on line 177 and assert it's there in line 175).